docs/core-concepts/packages.md

```diff
@@ -0,0 +1,170 @@
+Packages, Metapackages and Frameworks
+=====================================
+
+.NET Core is a platform made of NuGet packages. Some product experiences benefit from fine-grained definition of packages while others from course-grained. To accommodate this duality, the product is distributed as a fine-grained set of packages and then described in courser chunks with a package type informally called a "metapackage".
+
+Each of the .NET Core packages support being run on multiple .NET runtimes, represented as target frameworks. Some of those frameworks are traditional frameworks, like `net46`, representing the .NET Framework 4.6. Another set is new frameworks that can be thought of as "package-based frameworks", which establish a new model for defining frameworks. These package-based frameworks are entirely formed and defined as packages, forming a strong relationship between packages and frameworks.
+
+Packages
+========
+
+.NET Core is split into a set of packages, which provide primitives, higher-level data types, app composition types and common utilities. Each of these packages represent a single assembly of the same name. For example, [System.Runtime](https://www.nuget.org/packages/System.Runtime) contains System.Runtime.dll. 
+
+There are advantages to defining packages in a fine-grained manner:
+
+- Fine-grained packages can ship on their own schedule with relatively limited testing of other packages.
+- Fine-grained packages can provide differing OS and CPU support.
+- Fine-grained packages can have dependencies specific to only one library.
+- Apps are smaller because unreferenced packages don't become part of the app distribution.
```

Comment:
Your point is true, but it seems logically separate from "fine-grained packaging".  That's more of a library concept, IMO.